script.js

```diff
@@ -1,60 +1,49 @@
 const input = document.querySelector('#fruit');
-const suggestions = document.querySelector('.suggestions ul');
+const suggestionsList = document.querySelector('.suggestions ul');
```

Comment:
this is more nitpicking than an actual need, but keeping the habit of naming stuff correctly will save you tons in the future.